test/integration/targets/rds_param_group/aliases

```diff
@@ -0,0 +1,3 @@
+cloud/aws
+posix/ci/cloud/aws
+skip/python3
```

Comment:
The tests passed for me on python 3, so there appears to be no reason to skip them.